weave/trace_server/migrations/014_add_leaf_object_class.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE object_versions
```

Comment:
may want to update object_versions_stats_view as well

Reply:
i think you might need to also update `object_versions_stats_view`. it looks like the last update of `object_versions_stats` may have forgot this